middleman-core/lib/middleman-core/server_listener.rb

```diff
@@ -0,0 +1,72 @@
+require 'socket'
+require 'resolv'
+
+module Middleman
+  class ServerListener
+    private
+
+    attr_reader :https
```

Comment:
> No need to have this, just use @https in the code.

:smile: I know, but I follow a tip from Sandy Metz not to use instance variables, but use a method instead. But I think this is just a matter of style. Although I had a view uses cases where the tip proved to be useful. If you want me to use instance variables, I'm happy to change this.